modules/imap_folders/site.js

```diff
@@ -9,7 +9,10 @@ var folder_page_folder_list = function(container, title, link_class, target, id_
     $('.imap_folder_link', folders).addClass(link_class).removeClass('imap_folder_link');
     folder_location.prepend(folders);
     folder_location.show();
-    $('.'+link_class, folder_location).on("click", function() { return expand_folders_page_list($(this).data('target'), container, link_class, target, id_dest); });
+    if (subscription) {
+        expand_folders_page_list($('.'+link_class, folder_location).data('target'), container, link_class, target, id_dest, true);
```

Comment:
There is a duplicate trigger of this call - one here when subscription = true and the other one on line 15 when auto-clicking on the folders_subscription page resulting in 2 lists of folders with checkboxes.